lib/asyncapiSchemaFormatParser.js

```diff
@@ -80,12 +82,18 @@ function getValidator(version) {
  * 
  * @private
  * @param {Object} asyncapiSchema AsyncAPI specification JSON Schema
+ * @param {Object} version AsyncAPI version.
  * @returns {Object} valid JSON Schema document describing format of AsyncAPI-valid schema for message payload
  */
-function preparePayloadSchema(asyncapiSchema) {
+function preparePayloadSchema(asyncapiSchema, version) {
+  const payloadSchema = `http://asyncapi.com/definitions/${version}/schema.json`;
```

Comment:
why can't we grab it from `@asyncapi/specs` instead of remote?

sorry if that was answered somewhere already. Also, I do not remember really much from times when I created `asyncapiSchemaFormatParser.js`

just dereferencing something in the parser from remote is going to slow down parsing, or?

Reply:
We are using the local one from `@asyncapi/specs` 🙂 

It is only what the reference is called. Ajv automatically matches it with the schema inside `definitions` 🙂

Reply:
do we need http://asyncapi.com/definitions/ published before we merge?
tbh I'm not 100% sure how these $id work...to complicated for me 😄

Reply:
Not at all, the `$id`s when bundled together like so, do not need external access to the schemas 🙂 Because validation tools must look for the reference within before reaching externally. 

We don't need to expose them at all if everyone uses the bundled schemas, but say you want to use this one: https://github.com/asyncapi/spec-json-schemas/blob/master/definitions/2.4.0/asyncapi.json, it would need the schemas exposed on http://asyncapi.com/definitions/ as the schemas are not located within.

Does that make sense?